linux-subscriptions/src/list_linux_subscriptions_sample.py

```diff
@@ -0,0 +1,79 @@
+import boto3
+import base64
+import datetime
+import pprint
+import uuid
+from boto3.session import Session
+
+default_region = 'us-east-1'
+
+# Helper method to list linux subscriptions
+def list_linux_subscriptions():
+    lm_linux_subscriptions_client = get_client(default_region)
+    response = lm_linux_subscriptions_client.list_linux_subscriptions()
+    print('AWS License Manager Linux Subscriptions - ListLinuxSubscriptions API response:')
+    pprint.pprint(response)
+    return
+
+# Helper method to list linux subscription instances
+def list_linux_subscription_instances(FilterName, Condition, FilterValues):
+    lm_linux_subscriptions_client = get_client(default_region)
+    response = lm_linux_subscriptions_client.list_linux_subscription_instances(
+                    Filters=[
+                        {
+                            'Name': FilterName,
+                            'Operator': Condition, # 'Equal'|'NotEqual'|'Contains'
+                            'Values': FilterValues 
+                        }
+                    ]
+    )
+    while True:
+        print('AWS License Manager Linux Subscriptions - ListLinuxSubscriptions API response:')
+        pprint.pprint(response)
+        if "NextToken" in response:
+            next_token = response['NextToken']
+            response = lm_linux_subscriptions_client.list_linux_subscription_instances(
+                    Filters=[
+                        {
+                            'Name': FilterName,
+                            'Operator': Condition,
+                            'Values': FilterValues 
+                        }
+                    ]
+            )
+        else:
+            break
+    return
+
+# Helper method to get license manager linux subscription settings
+def get_linux_subscriptions_settings():
+    lm_linux_subscriptions_client = get_client(default_region)
+    response = lm_linux_subscriptions_client.get_service_settings()
+    print('AWS License Manager Linux Subscriptions - GetServiceSettings API response:')
+    pprint.pprint(response)
```

Comment:
Print - If org is not turned on - that we will be doing for it for this account only for these x,y,z home regions.